.github/workflows/build.yml

```diff
@@ -0,0 +1,14 @@
+name: Deploy if not a version bump PR
```

Comment:
Isn't this workflow the same as this one: [.github/workflows/build.yml ](https://github.com/nationalarchives/tdr-auth-utils/pull/49/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R1) Could they be added to the common library?

Reply:
It's not really worth it. You need to be able to call the deploy job inside this repo. We could have this in the common library but then we'd have to tell it which deploy job to run when it has finished and it all gets a bit messy as you have to do it through the rest API. 
You'd still need this file as well because the triggers on lines 2-5 need to be in the repo you're running it for.

Reply:
Fair enough.